crates/biome_js_analyze/tests/specs/nursery/noRestrictedElements/valid.options.json

```diff
@@ -0,0 +1,16 @@
+{
+	"$schema": "../../../../../../packages/@biomejs/biome/configuration_schema.json",
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"nursery": {
+				"noRestrictedElements": {
+					"level": "error",
+					"options": {
+						"elements": {}
```

Comment:
An empty list should be an error. This rule is useless with an empty list, which means we risk running a rule for nothing, affecting performance